homeassistant/components/assist_satellite/entity.py

```diff
@@ -26,18 +26,18 @@
 _CONVERSATION_TIMEOUT_SEC: Final = 5 * 60  # 5 minutes
 
 
+class AssistSatelliteEntityDescription(EntityDescription, frozen_or_thawed=True):
+    """A class that describes binary sensor entities."""
+
+
 class AssistSatelliteEntity(entity.Entity):
     """Entity encapsulating the state and functionality of an Assist satellite."""
 
-    entity_description = EntityDescription(
-        key="assist_satellite",
-        translation_key="assist_satellite",
-        entity_category=EntityCategory.CONFIG,
-    )
+    entity_description: AssistSatelliteEntityDescription
+    _attr_translation_key = "assist_satellite"
     _attr_has_entity_name = True
```

Comment:
These both need to be removed

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/voip/strings.json#L23-L37